ci/scripts/java_build.sh

```diff
@@ -23,9 +23,68 @@ source_dir=${1}/java
 cpp_build_dir=${2}/cpp/${ARROW_BUILD_TYPE:-debug}
 with_docs=${3:-false}
 
-mvn="mvn -B -DskipTests -Drat.skip=true -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn"
-# Use `2 * ncores` threads
-mvn="${mvn} -T 2C"
+if [[ "$(uname -s)" == "Linux" ]] && [[ "$(uname -m)" == "s390x" ]]; then
+  # Since some files for s390_64 are not available at maven central,
+  # download pre-build files from bintray and install them explicitly
+  mvn_install="mvn install:install-file"
+  wget="wget"
+  bintray_base_url="https://dl.bintray.com/apache/arrow"
+
+  bintray_dir="flatc-binary"
+  group="com.github.icexelloss"
+  artifact="flatc-linux-s390_64"
+  ver="1.9.0"
+  extension="exe"
+  target=${artifact}-${ver}.${extension}
+  ${wget} ${bintray_base_url}/${bintray_dir}/${ver}/${target}
+  ${mvn_install} -DgroupId=${group} -DartifactId=${artifact} -Dversion=${ver} -Dpackaging=${extension} -Dfile=$(pwd)/${target}
+
+  bintray_dir="protoc-binary"
+  group="com.google.protobuf"
+  artifact="protoc"
+  ver="3.7.1"
+  classifier="linux-s390_64"
+  extension="exe"
+  target=${artifact}-${ver}-${classifier}.${extension}
+  ${wget} ${bintray_base_url}/${bintray_dir}/${ver}/${target}
+  ${mvn_install} -DgroupId=${group} -DartifactId=${artifact} -Dversion=${ver} -Dclassifier=${classifier} -Dpackaging=${extension} -Dfile=$(pwd)/${target}
+  # protoc requires libprotoc.so.18
+  ${wget} ${bintray_base_url}/${bintray_dir}/${ver}/libprotoc.so.18
+  export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:$(pwd)
+
+  bintray_dir="protoc-gen-grpc-java-binary"
+  group="io.grpc"
+  artifact="protoc-gen-grpc-java"
+  ver="1.30.2"
+  classifier="linux-s390_64"
+  extension="exe"
+  target=${artifact}-${ver}-${classifier}.${extension}
+  ${wget} ${bintray_base_url}/${bintray_dir}/${ver}/${target}
+  ${mvn_install} -DgroupId=${group} -DartifactId=${artifact} -Dversion=${ver} -Dclassifier=${classifier} -Dpackaging=${extension} -Dfile=$(pwd)/${target}
+
+  bintray_dir="netty-binary"
+  group="io.netty"
+  artifact="netty-transport-native-unix-common"
+  ver="4.1.48.Final"
+  classifier="linux-s390_64"
+  extension="jar"
+  target=${artifact}-${ver}-${classifier}.${extension}
+  ${wget} ${bintray_base_url}/${bintray_dir}/${ver}/${target}
+  ${mvn_install} -DgroupId=${group} -DartifactId=${artifact} -Dversion=${ver} -Dclassifier=${classifier} -Dpackaging=${extension} -Dfile=$(pwd)/${target}
+  artifact="netty-transport-native-epoll"
+  extension="jar"
+  target=${artifact}-${ver}-${classifier}.${extension}
+  ${wget} ${bintray_base_url}/${bintray_dir}/${ver}/${target}
+  ${mvn_install} -DgroupId=${group} -DartifactId=${artifact} -Dversion=${ver} -Dclassifier=${classifier} -Dpackaging=${extension} -Dfile=$(pwd)/${target}
+
+  # Use `2 * ncores` threads
+  mvn_options="-T 2C -Dmaven.compiler.verbose=true"
```

Comment:
Can we remove `-Dmaven.compiler.verbose=true` and remove the `else` clause?

Reply:
sure, done